### PR TITLE
2083 feat use theme hook

### LINKED ISSUE #2083: [Feature?]: useTheme hook in v1 docs
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Summary 💡

migrate [beta docs](https://design.pluralsight.com/docs/reference/react-hooks/useTheme) useTheme doc to v1

### Examples 🌈

_No response_

### Motivation 🔦

so that users can reference Pando's useTheme hook in v1 docs

### PR DESCRIPTION
**Pull request by kathydlavoie-ps**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

closes #2083 

## What is the new behavior?

Adds useTheme theme & mode usage and reference information, as well as adds files for docs that will be referenced from the doc.

## Other information

https://github.com/pluralsight/pando/assets/146388897/e042a896-fa5f-4b39-b679-9ba65df2814d

### CODE REVIEW

File: docs/src/components/ReactLib/ThemeSelectButton.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { useTheme } from '@pluralsight/react'
 
 export default function ThemeSelectButton() {
-  const { mode, updateMode } = useTheme()
+  const { mode, updateMode } = useTheme('inkyBlue')
```
Comment (caseybaggz) [inline docs/src/components/ReactLib/ThemeSelectButton.tsx:4]: Did you have to add this because of Starlight?

**Review by caseybaggz — COMMENTED** [review]
Question - could we turn the first example into Live and have a button that toggles the theme to play with?

File: docs/src/components/ReactLib/ThemeSelectButton.tsx (hunk 1 above)
Comment (kathydlavoie-ps) [reply docs/src/components/ReactLib/ThemeSelectButton.tsx:4]: I realized that I wasn't actually setting auto-setting inkyBlue anywhere :thinking_face: I don't know if mine is cached and still working from when I manually hardcoded it before - I was just adding this to make sure other users actually have inkyBlue workin

File: docs/src/components/LiveCodeExamples/UseTheme/LiveUseTheme.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  }
+
+  return (
+    <>
+      <LiveCodeEditor
+        componentString={modeSwitchExample}
+        scope={scope}
+        noInline
+      />
+      {theme !== 'inkyBlue' && (
```
Comment (caseybaggz) [inline docs/src/components/LiveCodeExamples/UseTheme/LiveUseTheme.tsx:67]: This is a great spot for our handy `Show` component which improves both readability and performance. 😄

Comment (kathydlavoie-ps) [reply docs/src/components/LiveCodeExamples/UseTheme/LiveUseTheme.tsx:67]: awesome! great suggestion.

**Review by caseybaggz — APPROVED** [review]
:shipit:

**Pull request merged**